src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = /[.\-/]/;
```

Comment:
The use of a regular expression to define the old separator is not necessary since `fromFormat` will always provide the separator at the last index. You can directly access it using `fromFormat[fromFormat.length - 1]`.